src/quo2/components/avatars/group_avatar/style.cljs

```diff
@@ -3,12 +3,15 @@
 
 (defn container
   [{:keys [container-size customization-color theme]}]
-  {:width            container-size
-   :height           container-size
-   :align-items      :center
-   :justify-content  :center
-   :border-radius    (/ container-size 2)
-   :overflow         :hidden
-   :background-color (colors/theme-colors (colors/custom-color customization-color 50)
-                                          (colors/custom-color customization-color 60)
-                                          theme)})
+  (let [color (if (keyword? customization-color)
+                customization-color
+                (get colors/chat-color->customization-color customization-color :blue))]
```

Comment:
Maybe we should just add a default color in case its not keyword.
`chat-color->customization-color` mapping gives false impression that feature is working as expected.

Reply:
We properly need to implement group creation, with [new colors support ](https://www.figma.com/file/7KIYbhoqNGAIFonE0w9TDz/Messages-for-Mobile?type=design&node-id=3720-549489&mode=design&t=netiFoqRxp8bANEs-0), until then better to use fallback/default color.

Reply:
The status-go PR adds our customization list of colors to the initialization of the color parameter for a group/channel. But go returns hex values.

According to a suggestion made by @J-Son89, I am currently discussing with @cammellos the idea of using constants/keywords instead of hex strings for the sake of uniformity.

Reply:
> But go returns hex values.

Yes, but that's an old implementation that used old colors and I think randomly assigned one color to a particular chat on creation. New implementation with new colors has not yet implemented.

Reply:
I think it's better to unblock this issue to fix the issue on the milestone and report the findings in a comment on this PR so that we can properly introduce the correct implementation in the future when it's time for it.

Reply:
> The status-go PR adds our customization list of colors to the initialization of the color parameter for a group/channel. But go returns hex values.
> 
> According to a suggestion made by @J-Son89, I am currently discussing with @cammellos the idea of using constants/keywords instead of hex strings for the sake of uniformity.

something like https://github.com/status-im/status-go/blob/develop/multiaccounts/common/const.go?

Reply:
That's a way better approach, didn't notice it before.
Thank you so much Parvesh, will use it and ask for a another review.

Reply:
> I think it's better to unblock this issue to fix the issue on the milestone and report the findings in a comment on this PR so that we can properly introduce the correct implementation in the future when it's time for it.

Yes, I agree missing group icon doesn't look nice.
Can we use something like, until feature is implemented 

`(let [color (if (keyword? customization-color) customization-color :primary))]`

Reply:
Okay, let's hardcode the colors to blue till it's well implemented :)